src/main/java/uk/gov/justice/digital/delius/data/api/AdditionalIdentifier.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.justice.digital.delius.data.api;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@ApiModel(description = "Additional Identifier. \nCurrent values for the type are\n " +
```

Comment:
Not sure of the best way of conveying this info, but the swagger looks ok and is quite readable. In essence we are referencing reference data

Reply:
Suggest end point to retrieve data